scripts/deploy.js

```diff
@@ -0,0 +1,31 @@
+const hre = require("hardhat");
+
+async function main() {
+  let StorageContract = await hre.ethers.getContractFactory("TestEthStorageContractKZG");
+  let storageContract = await StorageContract.deploy(
+    [
+      17, // maxKvSizeBits
+      30, // shardSizeBits ~ 1G
+      2, // randomChecks
+      1, // minimumDiff
+      60, // cutoff
+      1024, // diffAdjDivisor
+      0, // treasuryShare
+    ],
+    0, // startTime
+    0, // storageCost
+    0, // dcfFactor
+    1048576, // nonceLimit
+    "0x0000000000000000000000000000000000000000", // treasury
+    0 // prepaidAmount
```

Comment:
Some practical parameters if a miner need to be tested with. FYI.  
const sc = await EthStorageContract.deploy(
    [
      17, // maxKvSizeBits
      21, // shardSizeBits
      2, // randomChecks
      1000000, // minimumDiff
      60, // targetIntervalSec
      40, // cutoff
      1024, // diffAdjDivisor
      100, // treasuryShare
    ],
    0, // startTime
    10000000000000, // storageCost
    340282365167313208607671216367074279424n, // dcfFactor
    1000000, // nonceLimit
    "0x0000000000000000000000000000000000000000", // treasury
    1000000000000, // prepaidAmount
    {
      gasPrice: 30000000000,
    }
  );